com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/logging/LogHelper.java

```diff
@@ -65,6 +65,6 @@ public static void setLogLevel(Level level) {
      * @param enable Set to true to enable verbose logging
      */
     public static void setVerboseLogging(boolean enabled) {
-        setLogLevel(enabled ? Level.CONFIG : Level.WARNING);
+        setLogLevel(enabled ? Level.FINE : Level.INFO);
```

Comment:
Default: INFO (See top of this file)
Verbose: Finer than default
Not verbose: Use same as default (INFO)